app/lib/features/onboarding/pages/login_page.dart

```diff
@@ -65,9 +67,8 @@ class _LoginPageState extends ConsumerState<LoginPage> {
             mainAxisAlignment: MainAxisAlignment.start,
             crossAxisAlignment: CrossAxisAlignment.stretch,
             children: [
-              const SizedBox(height: 12),
-              const LogoWidget(width: 150, height: 150),
-              const SizedBox(height: 20),
+              if (screenHeight > 650)
+                LogoWidget(width: imageSize, height: imageSize),
```

Comment:
we show now logo at all on smaller screens?

Reply:
The reason it that, user will not able to see `LOGIN` button on first visualisation on very small device screens. They need to scroll a bit so that I think better to hide upper side logo on very small devices so that necessary data are visible to user without scroll.